src/cache.rs

```diff
@@ -163,11 +163,13 @@ impl Cache {
     }
 
     /// Update the pages cache from the specified URL.
-    pub fn update(&self, archive_url: &str) -> Result<()> {
+    pub fn update(&self, archive_source: &str) -> Result<()> {
         self.ensure_cache_dir_exists()?;
 
+        let archive_url = format!("{}{}", archive_source, "tldr.zip");
```

Comment:
I think we can be more cautious here and add the `/`. If is not present in `archive_source`, the `archive_url` will be unexpected, and if it was already present, having `//` does not change the behavior 

```suggestion
        let archive_url = format!("{}/tldr.zip", archive_source);
```